profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -213,7 +213,7 @@ function ecas_login_page() {
 function ecas_logout($full_logout = FALSE) {
   global $user;
 
-  watchdog('user', 'Session closed for %name.', array('%name' => theme('placeholder', $user->name)));
+  watchdog('user', 'Session closed for %name.', array('%name' => format_username($user)));
```

Comment:
If using format_username, as documentation advises, check_plain should be used wrapping the returned string.

But I would just use $user->name, because it is more straight forward and also because it is the way it is used in function user_logout_current_user and for me to use same logic for same results is a plus.

Anyway, with or without format_username, we need a change in code here.